IntegrationTests/common/script/synth.tcl

```diff
@@ -23,7 +23,7 @@ set_property -name {STEPS.SYNTH_DESIGN.ARGS.MORE OPTIONS} -value {-mode out_of_c
 
 # Synthesis
 update_compile_order -fileset sources_1
-launch_runs synth_1 -jobs 4
+launch_runs synth_1 -jobs 64
```

Comment:
@aryd This may be excessive on machines with fewer CPU cores or less memory. I would recommend keeping the default at four or, better yet, getting the actual number of CPU cores from `nproc` using the `exec` command in Tcl.

```suggestion
set ncpus [exec nproc]
launch_runs synth_1 -jobs $ncpus
```

Reply:
Fixed